jobs/pr_gate/docker/rackhd.yml

```diff
@@ -2,18 +2,36 @@ apps:
   - script: index.js
     name: on-taskgraph
     cwd: /RackHD/on-taskgraph
+    out_file: /var/log/RackHD.log
```

Comment:
using the lower case filename `rackhd.log` (rather than `RackHD.log`) matches the naming convention with other files.